xcodebuild/build.go

```diff
@@ -131,9 +131,16 @@ func (c *CommandBuilder) SetTestPlan(testPlan string) *CommandBuilder {
 	return c
 }
 
-func (c *CommandBuilder) cmdSlice() []string {
+func (c CommandBuilder) cmdSlice() []string {
```

Comment:
Goland will complain that some function defined on the regular and some on the pointer for the struct. Can you update the changed functions to also use the pointer version in the method signature here and in the `ExportCommandModel` struct?

Reply:
Updated.